.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
```

Comment:
Nice! We should scope this to JS/TSX/HTML though, as Go and Kotlin have different formatting requirements.

Reply:
Oh gotcha! Sorry I thought you wanted the same setup for everything 😂 will update